packages/shared/src/hooks/useFeed.ts

```diff
@@ -159,22 +160,31 @@ export default function useFeed<T>(
     if (feedQuery.data) {
       newItems = feedQuery.data.pages.flatMap(
         ({ page }, pageIndex): FeedItem[] => {
-          const posts: FeedItem[] = page.edges.map(({ node }, index) => ({
-            type: 'post',
-            post: node,
-            page: pageIndex,
-            index,
-          }));
+          const posts: FeedItem[] = [];
+          if (onNewSquadPost) {
+            posts.push({
+              type: 'new_squad_post',
+              action: onNewSquadPost,
+            });
+          }
           if (adsQuery.data?.pages[pageIndex]) {
-            posts.splice(adSpot, 0, {
+            posts.push({
```

Comment:
Not sure if this is correct.
It should inject ads in certain spots.

Reply:
ahh, my bad. Fixed